### ISSUE TITLE
How to add software docs sites, en-masse?

### ISSUE THREAD

**Issue by NewAlexandria** [p4t0k/brave-goggles#1]
This is great, but there are thousands of docs sites which should be included in order to get complete results.   Thoughts?  Maybe scrape the package-manager sites for non-github URLs?

**Comment by p4t0k** [comment]
Yes, I agree, my goal was to extend the set of included sites over time. But what exactly do you mean by scraping the package-manager sites? Do you mean searching for URLs in packages' metadata? Could you provide me with some examples? Thanks.
